content/pages/sprint/2023/03-hsinchu.rst

```diff
@@ -0,0 +1,77 @@
+========================================
+Scisprint 2023 March at Hsinchu
+========================================
+
+:date: 2023-02-19 21:28
+:url: sprint/2023/03-hsinchu
+:save_as: sprint/2023/03-hsinchu.html
+
+Scisprint 2023 March at Hsinchu
```

Comment:
Same as line 2 above.
It should be aligned with the title in `index.rst` : `Scisprint 2023 March in Hsinchu`